tempo/serve/metadata.py

```diff
@@ -135,9 +135,25 @@ class Config:
         use_enum_values = True
 
 
+class StateTypes(Enum):
+    LOCAL = "LOCAL"
+    REDIS = "REDIS"
+
+
+class StateOptions(BaseModel):
+    state_type: Optional[StateTypes] = StateTypes.LOCAL
```

Comment:
For this one, would it make sense to use an actual import path instead of the `StateTypes` enum? Although, TBH that may be overkill for what we have now. If we had a feature request from users to override the State class, we could always think of changing it to something like [`PyObject`](https://pydantic-docs.helpmanual.io/usage/types/), but I think it's good for now.

Reply:
I think for the enum it may be overkill